nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ModifyBytes.java

```diff
@@ -66,13 +66,15 @@
             .required(true)
             .addValidator(StandardValidators.DATA_SIZE_VALIDATOR)
             .defaultValue("0 B")
+            .expressionLanguageSupported(true)
             .build();
     public static final PropertyDescriptor END_OFFSET = new PropertyDescriptor.Builder()
             .name("End Offset")
```

Comment:
When I saw _end-offset_  i got confused since the next offset is the end offset, but description states its _Number of bytes removed at the end of the file_. This means it's not an offset at all.
I know this is not part of the immediate issue, but at least consider raising an improvement JIRA where this could be fixed. Obviously we can't change the _name_, so we'll keep it, but we can add _displayProperty(..)_